addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -317,11 +321,11 @@ function factory(dependencies) {
                 audioTrack.addEventListener('ended', async () => {
                     // this mostly happens when the user retracts microphone permission.
                     await this.async(() => this.updateLocalAudioTrack(false));
-                    this.currentRtcSession.updateAndBroadcast({ isMuted: true });
+                    this.currentRtcSession.updateAndBroadcast({ isSelfMuted: true });
                     await this.async(() => this._updateLocalAudioTrackEnabledState());
                 });
-                this.currentRtcSession.updateAndBroadcast({ isMuted: false });
-                audioTrack.enabled = !this.currentRtcSession.isMuted && this.currentRtcSession.isTalking;
+                this.currentRtcSession.updateAndBroadcast({ isSelfMuted: false });
+                audioTrack.enabled = !this.currentRtcSession.isMute && this.currentRtcSession.isTalking;
```

Comment:
Slightly out of scope of PR, but I see this expression more than once and it'd be appropriate to define a new concept resulting from this expression. Something like `isEffectivelyTalking` or something like that.

The documentation of field `RtcSession/isTalking` lacks subtle information that talking can happen also in mute state. BTW, the modelling looks strange that we could have `isTalking` while `isMute`. `isMute` should enforce not is talking, because being mute contradicts any possibility to talk.

Maybe the intend of `isTalking` is more about modelling the state "attempting to talk"? This could be meaningful in modelling, like having UI feedback that it fails talking due to mute state. That's actually what Discord does (small popover with text "You are muted!").